readme.md

```diff
@@ -120,6 +121,7 @@ Click the type names for complete docs.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.
 - [`Simplify`](source/simplify.d.ts) - Flatten the type output to improve type hints shown in editors.
+- [`Jsonify`](source/jsonify.d.ts) - Transform a type to a type that is assignable to the `JsonValue` type
```

Comment:
```suggestion
- [`Jsonify`](source/jsonify.d.ts) - Transform a type to one that is assignable to the `JsonValue` type
```

How about this? Felt like too many "type" words.

Reply:
The description should also end in a `.`

Reply:
See 0f760c94deb510f0829ec4dd90e3adb0909c8356